src/styles/index.scss

```diff
@@ -1,3 +1,117 @@
-body {
+$width: 200px;
+$color: #00acdc;
+$color1: #060b35;
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+.main {
+  width: #{$width};
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  box-sizing: border-box;
```

Comment:
add it to * selector

Reply:
![image](https://github.com/mate-academy/layout_product-cards/assets/55200530/762b8686-3030-4bef-b9a5-946eaccff72f)

Reply:
https://github.com/mate-academy/layout_product-cards/pull/3622/files#r1337438248
not fixed